tests/Test.js

```diff
@@ -418,4 +420,11 @@ const JSON_WINDOWSTATUS = Test.JSON_WINDOWSTATUS = {
     [WindowStatus.KEY_STATE]: Test.JSON_WINDOW_STATE,
 };
 
-module.exports = Test;
\ No newline at end of file
+const GENERAL_TEMPERATURE_UNIT = Test.GENERAL_TEMPERATURE_UNIT = TemperatureUnit.CELSIUS;
+const GENERAL_TEMPERATURE = Test.GENERAL_TEMPERATURE = new Temperature()
+    .setUnit(Test.GENERAL_TEMPERATURE_UNIT)
+    .setValueParam(Test.GENERAL_NUMBER);
+
+const JSON_TEMPERATURE = Test.JSON_TEMPERATURE = GENERAL_TEMPERATURE.getParameters();
+
```

Comment:
You can instantiate the `ClimateData` objects (`GENERAL_CLIMATE_DATA` and `JSON_CLIMATE_DATA`) around line 429.  It would be something similar to lines 402-406 for the Window State setup (`GENERAL_WINDOW_STATE` and `JSON_WINDOW_STATE`)

Reply:
Fixed in d1e6617